src/pages/docs/step-ca/policies.mdx

```diff
@@ -0,0 +1,295 @@
+
+#### Examples
+
+Type    | Rule              | Matches                               | Non-Matches
+--------|:-:                |:-:                                    |:-:
+dns     |host.example.com   |host.example.com                       |differenthost.example.com, sub.host.example.com
+dns     |*.example.com      |host.example.com, www.example.com      |example.com, sub.host.example.com
+
+
+### IP Addresses 
+
+Rules for evaluating IP addresses are evaluated as follows:
+
+- A single IP rule (e.g. `192.168.0.30`, `::1`) will match exactly that IP address.
+- A CIDR rule (e.g. `192.168.0.0/24`, `2001:0db8:85a3::8a2e:0370:7334/120`) matches all IP addresses included in the range.
+- So-called `IPv4-in-IPv6` addresses will be matched using rules for either IPv6 or IPv4. This is because Go doesn't distinguish these.
+
+#### Examples
+
+Type    | Rule                              | Matches                       | Non-Matches
+--------|:-:                                |:-:                            |:-:
+ip      |192.168.0.1                        |192.168.0.1                    |192.168.0.30, 10.0.0.1
+ip      |192.168.0.0/24                     |192.168.0.1, 192.168.0.10      |192.168.20.1, 10.0.0.1
+ip      |::1                                |::1                            |::2
+ip      |2001:0db8:85a3::8a2e:0370:7334/120 |2001:0db8:85a3::8a2e:0370:7334 |3001:0db8:85a3::8a2e:0370:7334
+
+
+### Email Addresses 
+
+Rules for evaluating email addresses are evaluated as follows:
+
+- Email domain names are normalized into their [internationalized representation (RFC 5890)](https://datatracker.ietf.org/doc/html/rfc5890).
+- A specific email address (e.g. `jdoe@example.com`) will match exactly that email address.
+- A rule without a specific local part (e.g. `@example.com`) will match any email address ending in `@example.com`.
+
+It is not possible to configure a wildcard subdomain.
+You'll need to configure multiple email rules in case you need to match email addresses using different subdomains.
+
+#### Examples
+
+Type        | Rule                      | Matches                                   | Non-Matches
+------------|:-:                        |:-:                                        |:-:
+email       |jdoe@example.com           |jdoe@example.com                           |janedoe@example.com
+email       |@example.com               |jdoe@example.com, janedoe@example.com      |jdoe@www.example.com, jdoe@somehost.com
+
+
+
+### URIs
+
+Rules for evaluating URIs are evaluated as follows:
+
+- URI domains are normalized into their [internationalized representation (RFC 5890)](https://datatracker.ietf.org/doc/html/rfc5890).
+- URI rules must contain an authority part and cannot contain IP addresses, in correspondence with [RFC 5280](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.10).
+- URI schemes are not matched. When configuring an URI rule, the scheme is ignored.
```

Comment:
Hmm. Could this be considered subtly insecure in some way?

Reply:
Yes, I think people may be slightly surprised by this and it could result in a security issue. I remember we discussed a more complete form of URI policy evaluation that would include URNs (a.o.), but decided not to implement it at first. This is probably a good candidate for the skull and bones? 🙂 

It may not be that much work to check the scheme if it was present in in the rule, but I'll need to check this. The harder thing is determining the broader semantics of this: is a scheme optional in the rule? If not, do we need a wildcard for the scheme? Do we also want to support other parts of the URI, like the path, query and fragment?